### PR TITLE
Clean up unused methods in utils

### PR DESCRIPTION
**Pull request by adk9**
* Use utils:: namespace for utility functions
* Clean up unused methods in utils
* Add `util::DumpNGGraph` to dump nGraph graph

### CODE REVIEW

**Review by kanvi-nervana — CHANGES_REQUESTED** [review]

File: ngraph_bridge/ngraph_rewrite_pass.cc [hunk 1]
```diff
@@ -79,15 +79,16 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
     // runs of this pass.
     int idx = FreshIndex();
 
+    tensorflow::Graph* graph = options.graph->get();
```
[the remark this answers is not in the document]
Comment (adk9) [reply ngraph_bridge/ngraph_rewrite_pass.cc:82]: No, it's mutated in-place.

File: ngraph_bridge/ngraph_utils.cc [hunk 2]
```diff
@@ -367,68 +223,35 @@ void MemoryProfile(long& vm_usage, long& resident_set) {
...
   }
-  // If we have a "main" graph, dump that.
-  if (options.graph != nullptr) {
-    auto pbtxt_filename = PbtxtFilename(filename_prefix, idx);
-    NGRAPH_VLOG(0) << "Dumping main graph to " << pbtxt_filename;
-    GraphToPbTextFile(options.graph->get(), pbtxt_filename);
-  }
 
-  // If we have partition graphs (we shouldn't), dump those.
-  if (options.partition_graphs != nullptr) {
```
[the remark this answers is not in the document]
Comment (adk9) [reply ngraph_bridge/ngraph_utils.cc:412]: No, the caller of this method will have to dump partitions/sub-graphs if any. In which case will we need to dump partitions?

File: ngraph_bridge/ngraph_utils.cc [hunk 3]
```diff
@@ -36,103 +36,17 @@ using namespace std;
...
-// Write into this ng_tensor from tf_tensor
-void WriteNGTensor(shared_ptr<ngraph::runtime::Tensor> ng_tensor,
-                   Tensor* tf_tensor) {
-  NG_TRACE("Tensor Write H2D", "", "");
-  void* tf_src_ptr = (void*)DMAHelper::base(tf_tensor);
-  ng_tensor->write(tf_src_ptr, ng_tensor->get_element_count() *
-                                   ng_tensor->get_element_type().size());
-}
-
-void SummarizeOp(OpKernelConstruction* ctx, std::ostream& out) {
```
[the remark this answers is not in the document]
Comment (adk9) [reply ngraph_bridge/ngraph_utils.cc:123]: They aren't being used anywhere.

File: ngraph_bridge/ngraph_utils.cc (hunk 2 above)
[the remark this answers is not in the document]
Comment (kanvi-nervana) [reply ngraph_bridge/ngraph_utils.cc:412]: We probably won't see partitions, that happens after POST_REWRITE

**Pull request merged**